env.dev.sh

```diff
@@ -3,4 +3,6 @@ export MYSQL_USERNAME=root
 export MYSQL_PASSWORD=root
 export MYSQL_DATABASE=noah
 export MYSQL_HOST=0.0.0.0
-export MYSQL_PORT=3306
\ No newline at end of file
+export MYSQL_PORT=3306
+export KALEYRA_API_KEY=A9430fc99b19f0d94f86222c84ac38f54
```

Comment:
This is a public repo, please do not put API keys in checked-in files.

Reply:
I completely forgot about that. Sorry. Deleting this key and creating a new one.